_plugins/docs_generator.rb

```diff
@@ -145,7 +145,7 @@ def build_with_sphinx(repo_name, repo_path, repo_data)
     output_path = Pathname.new(File.join(repo_path, '_build'))
     FileUtils.rm_r(output_path) if File.directory? output_path
     FileUtils.makedirs(output_path)
-    command = "python3 -m sphinx -b json -c #{repo_path} #{input_path} #{output_path}"
+    command = "LC_ALL=C.UTF-8 python3 -m sphinx -vvv -b json -c #{repo_path} #{input_path} #{output_path}"
```

Comment:
Using an explicit encoding makes sense. Adding the verbose is good as long as it won't blow up our console output too much. Though since this is just the redirects I think that's fine.